packages/@guardian/source-react-components/src/brand/SvgGuardianBestWebsiteLogo.tsx

```diff
@@ -0,0 +1,59 @@
+				fill={textAccentColor}
+			/>
+		</svg>
```

Comment:
I know this hasn't been done in the other brand logos yet, but there are a few changes that will help make this more accessible to assistive technology users:

- add `focusable={false}` to the `<svg>` props, to prevent this image from erroneously receiving focus (an IE11 bug)
- add `aria-hidden={true}` to the `<svg>` props, to prevent the markup of this image from being announced by screen readers
- add a visually `<span>` as a sibling with alt text. e.g.

```
	<>
		<svg />
		<span css={css`${visuallyHidden}`}>
			The Guardian: news website of the year
		</span>
	</>
```

Reply:
Thanks for this Simon! I've added this to the new logo and taken the liberty of adding them to the existing logos as well.